manager/src/test/java/com/redhat/service/bridge/manager/ProcessorServiceTest.java

```diff
@@ -126,40 +125,27 @@ private BaseAction createKafkaAction() {
         return a;
     }
 
-    @Test
```

Comment:
Why did you delete these tests?

Reply:
`bridgeNotActive` and `bridgeDoesNotExist` checks are moved to controllers. `processorService.createProcessor` functions would call only when bridge is active/exists.

Reply:
ok

Reply:
> `bridgeNotActive` and `bridgeDoesNotExist` checks are moved to controllers.

I think we should consider each service as independent and not always assume a certain behavior by the caller.
I think these tests must be restored.
Also...I don't see where these checks have been moved in this PR.

Reply:
@kostola 
This change had already been discussed with Luca and Rob in above review comment.
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/422#discussion_r823957472

These changes are now moved to API's.
https://github.com/vaibhavjainwiz/sandbox/blob/91ef48b69884cc3532ca9f9e06cf0ac79f3f43ac/manager/src/main/java/com/redhat/service/bridge/manager/api/user/ProcessorsAPI.java#L125